submissions/xsorter/port-sniffer/program.js

```diff
@@ -0,0 +1,95 @@
+const net = require('net');
+const dns = require('dns');
+const args = require('minimist')(process.argv.slice(2));
+const range = findPortsRange();
+
+const ports = {
+  firstPort: +range[0],
+  lastPort: +range[1] ? +range[1] : +range[0],
+  openedPorts: []
+};
+
+const openedPortCheck = (host, port, checkCallback) => {
+  const socket = net.createConnection(port, host);
+  const time = socket.setTimeout(300, () => {
+    socket.destroy();
+    checkCallback(false);
+  });
+
+  socket.on('connect', () => {
+    clearTimeout(time);
+    socket.destroy();
+    process.stdout.write('.');
```

Comment:
Dont use process.stdout.write
Prefet to use console.log / warn

Reply:
Done, changed to console.log